staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/rules/rules.go

```diff
@@ -50,6 +53,22 @@ func exactOrWildcard(items []string, requested string) bool {
 	return false
 }
 
+var namespaceResource = schema.GroupVersionResource{Group: "", Version: "v1", Resource: "namespaces"}
+
+func (r *Matcher) scope() bool {
+	if r.Rule.Scope == nil || *r.Rule.Scope == v1beta1.AllScopes {
+		return true
+	}
+	switch *r.Rule.Scope {
+	case v1beta1.NamespacedScope:
+		return r.Attr.GetResource() != namespaceResource && r.Attr.GetNamespace() != metav1.NamespaceNone
```

Comment:
Namespace of Namespace should be empty string?  Or is this an admission quirk?  Deserves a comment.

Elsewhere (in resource builder) namespace of namespace is empty.

Reply:
request attributes populate namespace attribute based on path segments, so an individual namespace populates that attribute. will comment on the method